addons/wowl/static/src/services/action_manager/action_manager.ts

```diff
@@ -34,6 +34,7 @@ export type ActionRequest = ActionId | ActionXMLId | ActionTag | ActionDescripti
 export interface ActionOptions {
   clearBreadcrumbs?: boolean;
   viewType?: ViewType;
+  resID?: number;
```

Comment:
convention in our branch: resId, I think

Reply:
Done, @aab-odoo